test/Cardano/BM/Test/Configuration.lhs

```diff
@@ -249,7 +258,7 @@ unitConfigurationParsedRepresentation = do
             , "    - monitor: ((time > (23 s)) Or (time < (17 s)))"
             , "    - actions:"
             , "      - AlterMinSeverity \"chain.creation\" Debug"
-            , "    '#aggregation.critproc.observable':"
+            , "    ! '#aggregation.critproc.observable':"
```

Comment:
this probably has to do with an outdated libyaml. let's find out..

Reply:
Without this `!` id soesn't work.. :-(

Reply:
right, we have to recreate consistency among compilation with nix, cabal and stack.